static/app/views/dataExport/dataDownload.tsx

```diff
@@ -206,12 +211,12 @@ class DataDownload extends DeprecatedAsyncView<Props, State> {
           <p>
             {t("That link won't last forever — it expires:")}
             <br />
-            {this.renderDate(dateExpired)}
+            {renderDate(dateExpired)}
           </p>
-          {this.renderOpenInDiscover()}
+          {renderOpenInDiscover()}
           <p>
             <small>
-              <strong>SHA1:{checksum}</strong>
+              <strong>{tct('SHA1: ', {checksum})}</strong>
```

Comment:
I dont think this will ever translate to anything since SHA1 is just the name of the hashing algorithm